src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -80,7 +128,12 @@ public RoadAccident getAccidentByIndex7(String index){
 
 
     public RoadAccident getAccidentByIndex(String index){
-        return null;
+        RoadAccident roadAccident = roadAccidentList.stream()
+                .filter( rdAcc->{
+                    return rdAcc.getAccidentId().equals(index) ;
+                }).findFirst().get();
```

Comment:
findFirst return Optional object. 

Optional.get() will throw exception if no match found. 

So I recommend to use orElse method instead. 

.orElse(null) - it will try to get value first, and return null if value not found.